rdmd_test.d

```diff
@@ -88,65 +91,65 @@ void runTests()
     string forceSrc = tempDir().buildPath("force_src_.d");
     std.file.write(forceSrc, `void main() { pragma(msg, "compile_force_src"); }`);
 
-    res = execute([rdmdApp, compilerSwitch, forceSrc]);
+    res = execute([rdmdApp, compilerSwitch, modelSwitch, forceSrc]);
```

Comment:
You can just break the array and concatenate sub-arrays.

Anyway I looked and `rdmdApp` always occurs together with `compilerSwitch`, so I'd refactor those into an array, so that the invocation looks like `execute(rdmdArgs ~ [...])`